### PR TITLE
Update CHANGELOG for Loris 24.0

### PR DESCRIPTION
**Pull request by CamilleBeau**
## Brief summary of changes
This PR updates the changelog for relevant PRs included in 24.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,8 +25,12 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Addition of autoSelect prop to React SelectElement allows for auto-selection of only available select option (PR #6156)
 - An `AcquisitionDate` field has been added to the `files` table (PR #6892)
 - The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0 (PR #7496).
+- Pending accounts in Dashboard are now includes DDC users (PR #7054)
```
Comment (ridz1208) [inline CHANGELOG.md:28]: review your sentence

**Review by ridz1208 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,8 +25,12 @@ requesting a new account and will be displayed in the User Accounts module (PR #
...
+- Subproject filter added to Behavioural QC module (PR #7430)
 #### Bug Fixes
 - The default value of the `ScannerID` field of the `mri_protocol` table is now `NULL` instead of `0`. This means that if a protocol is valid on all the study's scanners, then `ScannerID` of the protocol should be set to `NULL` (PR #7496)
+- In Candidate Parameters, fix caveat emptor flag being editable without a permission to edit candidate profiles(PR #7047)
+- Candidate library now allows a null sex in the select() function to accommodate scanner candidates. This prevents an error from being thrown in the candidate parameters module. (PR #7058)
```
Comment (ridz1208) [inline CHANGELOG.md:33]: not necessary, seems like a bug introduced by the same version

Comment (CamilleBeau) [reply CHANGELOG.md:33]: Are you talking about line 32 or 33 or both?

Comment (ridz1208) [reply CHANGELOG.md:42]: if the bug on line 32 existed in version 23, keep it, if it was introduced and fixed in 24, remove it

Comment (CamilleBeau) [reply CHANGELOG.md:42]: Looks like it did not exist in 23 I will remove

File: CHANGELOG.md [hunk 3]
```diff
@@ -33,9 +33,12 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - An `AcquisitionDate` field has been added to the `files` table (PR #6892)
 - Data_entry_completion_status given its own column in flag, and renamed to Required_elements_completed (PR #6876)
 - The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0 (PR #7496).
+- Pending accounts in Dashboard now includes DDC users (PR #7054)
```
Comment (ridz1208) [inline CHANGELOG.md:36]: DDC ?

File: CHANGELOG.md (hunk 2 above)
Comment (ridz1208) [reply CHANGELOG.md:41]: did the bug on line 41 exist in 23 ? if not remove it as well

**Review by ridz1208 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (CamilleBeau) [reply CHANGELOG.md:36]: OH. There was a different typo but I didn't catch this one

**Comment by driusan** [comment]
@ridz1208 conflicts

**Pull request merged**